chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CacheInitializor.java

```diff
@@ -0,0 +1,21 @@
+package gov.healthit.chpl.domain;
```

Comment:
The "domain" package contains objects that we use to communicate back and forth with the front end - the objects that get serialized into JSON. That doesn't seem like the right place for this class.